seek-camera/sys/build.rs

```diff
@@ -60,7 +60,11 @@ fn main() -> Result<()> {
         // We will allow users to specify the SDK path if they don't want to have it system installed.
         // For example, if on mac, you can use this to get around the fact that the seek sdk only
         // works on windows and linux.
-        let sdk_path = env::var("SEEK_SDK_PATH").ok().map(PathBuf::from);
+        let sdk_path = env::var("SEEK_SDK_OVERRIDE")
```

Comment:
This change should be reverted, as the env var that is used to override the flake input, and the env var that locates the actual sdk, are semantically different.

You can see for example how we actually use the flake input to then compute the value of the `SEEK_SDK_PATH` in the `flake.nix` [here](https://github.com/worldcoin/orb-software/blob/270fd92676799fce3af65dc88a56e7a76ecef992/flake.nix#L121) and [here](https://github.com/worldcoin/orb-software/blob/270fd92676799fce3af65dc88a56e7a76ecef992/flake.nix#L38).

Lets make this clear by naming the env var in the .envrc and CONTRIBUTING.md `FLAKE_OVERRIDE_SEEK_SDK` so its clear that it is only related to the nix flake, rather than the actual rust code.

Reply:
Yep, I was too fast in implementing without properly looking at full env var resolve pipeline. Now it should be consistent.